index.bs

```diff
@@ -3705,6 +3688,27 @@ To <dfn>trigger event-level attribution</dfn> given an [=attribution trigger=] |
              ("<code>[=trigger debug data type/trigger-event-low-priority=]</code>", |report|)).
 
     </dl>
+1. Let |rateLimitRecord| be a new [=attribution rate-limit record=] with the items:
```

Comment:
To be consistent, shall we move the storage-limit check in line 3676 here as well, and we just break for `add-new-report`.

Reply:
Done -- is this what you had in mind?

Reply:
yes, thanks.